ai-services/assets/applications/RAG/templates/vllm-server.yaml.tmpl

```diff
@@ -23,11 +23,17 @@ spec:
         type: Directory
   containers:
     - name: instruct
-      image: icr.io/ibmaiu_internal/ppc64le/dd2/spyre-vllm:v1.0.1
-      command: ["/bin/sh", "-c"]
-      args: [
-          "/home/senuser/container-scripts/vllm_start.sh --served-model-name ibm-granite/granite-3.3-8b-instruct"
-      ]
+      image: icr.io/ibmaiu_internal/ppc64le/dd2/rhaiis:3.2.5-ci_236
```

Comment:
Let us have a placeholder jira issue to replace this image with the production RHAIIS vllm image